src/importlinter/contracts/forbidden.py

```diff
@@ -7,23 +7,22 @@
 class ForbiddenContract(Contract):
     """
     Forbidden contracts check that one set of modules are not imported by another set of modules.
-
     Indirect imports will also be checked.
-
     Configuration options:
-
         - source_modules:    A list of Modules that should not import the forbidden modules.
         - forbidden_modules: A list of Modules that should not be imported by the source modules.
         - ignore_imports:    A set of DirectImports. These imports will be ignored: if the import
                              would cause a contract to be broken, adding it to the set will cause
                              the contract be kept instead. (Optional.)
+        - allow_indirect_imports: Bool:r indirect imports are allowed. (Default: False.)
     """
 
     type_name = "forbidden"
 
     source_modules = fields.ListField(subfield=fields.ModuleField())
     forbidden_modules = fields.ListField(subfield=fields.ModuleField())
     ignore_imports = fields.SetField(subfield=fields.DirectImportField(), required=False)
+    allow_indirect_imports = fields.StringField(required=False)
```

Comment:
NIP: maybe let's add `BoolField` to `importlinter.domain.fields` module, it's one of the most commonly used types

Reply:
Good idea! Not essential for this PR though.